Makefile

```diff
@@ -131,7 +135,8 @@ deploy: # Deploy/redeploy previously built REANA cluster.
 	@echo -e "\033[1;32m[$$(date +%Y-%m-%dT%H:%M:%S)]\033[1;33m reana:\033[0m\033[1m make deploy\033[0m"
 	source ${HOME}/.virtualenvs/${VENV_NAME}/bin/activate && \
 	minikube docker-env --profile ${MINIKUBE_PROFILE} > /dev/null && eval $$(minikube docker-env --profile ${MINIKUBE_PROFILE}) && \
-	helm dep update helm/reana && helm ls | grep -q reana && helm uninstall reana || \
+	kubectl delete secrets --all && \
+	helm dep update helm/reana && helm ls | grep -q ${TRUNC_REANA_COMPONENT_PREFIX} && helm uninstall ${TRUNC_REANA_COMPONENT_PREFIX} || \
```

Comment:
I saw troubles in production mode with uninstalling, i.e. the uninstall went fine but left some crumbs behind, so it was not possible to do this:

```console
$ DEMO=r-d-r-roofit make ci
$ DEMO=r-d-r-roofit make ci
```

Reply:
Can you also amend `reana-dev setup-environment`?  It does not take into account profile... If we go for unification, we can actually:

- detect Python venv this command runs in
- automatically add `--profile venv` for minkube commands
- actually, since `reana-dev` is used by developers and for Minikube, we could even run `docker-env` command perhaps?

Personally I've been using:

```console
$ eval $(minikube docker-env) && eval $(reana-dev setup-environment)
```

which now obviously have to be different... So I wonder if we do:

```console
$ workon reana
$ eval $(reana-dev setup-environment)
$ workon nightlybuilds
$ eval $(reana-dev setup-environment)
```

This may be more comfortable than having to add `--profile` to each command?!  Talk IRL